src/test/suite/utils/orgUtils.test.ts

```diff
@@ -165,17 +165,20 @@ suite('Org Utils Test Suite', () => {
                 {
                     editableForNew: true,
                     editableForUpdate: true,
-                    label: 'Name'
+                    label: 'Name',
```

Comment:
We could technically remove editableForNew/Update and this label value, because we thought we were going to use it and so far dont think we need to.

We still need to do some testing. It appears that the base LWC components used in our view/edit/create templates checks these values for us.

However, if we include "Owner", for example which has editableForNew=false and editableForUpdate=false, on the edit and create LWCs, it will still be shown, but will be disabled. Functionality still works fine. I think for the CREATE case, this may be odd because it will be a blank field shown, but still disabled. For the EDIT case, it still makes sense to show the value.